Gemfile

```diff
@@ -32,6 +32,12 @@ gem "puma", "~> 2.16.0"
 
 gem 'pg', platforms: :ruby
 
+group :production, :staging do
```

Comment:
This bit of the gemfile was just copied over from Caseflow. Looks like we need at least `ruby-oci8` in feedback because the `authenticate_vacols` method in the authentication service uses it. @orischwartz or @shanear, can you confirm?